examples/simple.c

```diff
@@ -59,17 +59,16 @@ bool process_loop(pid_t pid, CProcess process, void *data) {
 int main() {
     CSystem system = sysinfo_init();
     sysinfo_refresh_all(system);
-    printf("total memory:         %ld\n", sysinfo_get_total_memory(system));
-    printf("free memory:          %ld\n", sysinfo_get_free_memory(system));
-    printf("used memory:          %ld\n", sysinfo_get_used_memory(system));
-    printf("total swap:           %ld\n", sysinfo_get_total_swap(system));
-    printf("free swap:            %ld\n", sysinfo_get_free_swap(system));
-    printf("used swap:            %ld\n", sysinfo_get_used_swap(system));
-    printf("networks received:    %ld\n", sysinfo_get_networks_received(system));
-    printf("networks transmitted: %ld\n", sysinfo_get_networks_transmitted(system));
-    unsigned int len = 0, i = 0;
+    printf("total memory:         %ld\n", sysinfo_total_memory(system));
+    printf("free memory:          %ld\n", sysinfo_free_memory(system));
+    printf("used memory:          %ld\n", sysinfo_used_memory(system));
+    printf("total swap:           %ld\n", sysinfo_total_swap(system));
+    printf("free swap:            %ld\n", sysinfo_free_swap(system));
+    printf("used swap:            %ld\n", sysinfo_used_swap(system));
+    printf("networks received:    %ld\n", sysinfo_networks_received(system));
+    printf("networks transmitted: %ld\n", sysinfo_networks_transmitted(system));    unsigned int len = 0, i = 0;
```

Comment:
Nit: please put back the declaration of the variables on another line like it was before.

Reply:
Oops, copy/paste error, thank you!